### DISCUSSION TITLE
Cant change column's " Use as table display column"

### DISCUSSION THREAD

**Discussion by skorphil** [Budibase/budibase#9538] [Help] [answered]
Hi, every time i'm trying to change "Use as table display column" it reverts to a previous option after saving. Either i'm swithing it on or off
<img width="386" alt="Screenshot 2023-02-05 at 13 52 06" src="https://user-images.githubusercontent.com/6762009/216812277-d708873e-5ceb-430a-991e-4771c5efd915.png">

**Comment by rs911** [comment]
I'm having the same issue. I am using a Date type column in Postgres. 
I tried turning off the 'Use as table display column' for the other column. It reverts back.
Then tried turning on this option in another column. It reverts back. No changes happen.

**Comment by skorphil** [comment, marked as answer]
@rs911 There is the same issue described here - https://github.com/Budibase/budibase/discussions/9534 
I explained my temporary workaround there. Which is not as good though

**Reply by melohagan** [reply]
This has been fixed in the latest version!
